apps/allocations/app/store/entry.js

```diff
@@ -40,15 +42,15 @@ const loadEntryData = async (addr, addressBook) => {
 
   return addressBookApp
     .getEntry(addr)
-    .first()
-    .map(
+    .pipe(first())
+    .pipe(map(
```

Comment:
Could move this into the previous pipe, if the map seems too large for readability could be a custom operator. I think this mapping functionality is re-used. Neither is critical though.

Reply:
Will change